2_adventures/001/phonbopit/04_guessing_game/guessing_game.rb

```diff
@@ -0,0 +1,28 @@
+secret_number = rand(100) + 1
+
+puts "Welcome to Guessing Game :)"
+puts "-" * 20
+
+won = false
+
+5.times do
+	print "Choose a number between 1 - 100  ===> "
+	guess = gets.chomp.to_i
```

Comment:
[Is it ever necessary to use `.chomp` with `.to_i`](http://stackoverflow.com/q/20598679/485864)?